src/badges.js

```diff
@@ -50,6 +51,17 @@ const commonMinStyle = `
     </style>
 `
 
+const googleStyle = `
+    <style>
+    .scite-badge {
+      display: block;
+      width: min-content;
+      margin-top: -1rem;
```

Comment:
This is causing problems for me....
<img width="660" alt="Screen Shot 2021-11-30 at 1 53 12 PM" src="https://user-images.githubusercontent.com/15069938/144101217-189d0614-d8b4-4919-9800-830e9cca1101.png">

Reply:
Hmm.. maybe it is different for different results or something, I will just remove sad